maintainers/maintainer-list.nix

```diff
@@ -6570,6 +6570,12 @@
     githubId = 2029444;
     name = "James Kent";
   };
+  kephasp = {
+    email = "pierre@nothos.net";
+    github = "kephas";
+    githubId = 762421;
+    name = "Pierre Thierry";
+  };
```

Comment:
This needs to be in its own commit. See contributing

Reply:
I didn't find the relevant part in `CONTRIBUTING.md` but I imitated previous commits on that file for the commit message.